packages/apps-engine/src/server/runtime/deno/LivenessManager.ts

```diff
@@ -10,7 +10,7 @@ const defaultOptions: LivenessManager['options'] = {
     pingRequestTimeout: 10000,
     pingFrequencyInMS: 10000,
     consecutiveTimeoutLimit: 4,
-    maxRestarts: 3,
+    maxRestarts: Infinity,
```

Comment:
Is this safe? If an app restarted more than 10 times, should we continue trying to restart?

Reply:
Not really, but until we have a place where we can show the restart information it will just cause confusion as to why the app is not working properly. Mainly because a simple "enable" call will not make the engine create a new subprocess for the app

Reply:
Is there any chance of deno being "restarted" wrongly? 

I ask because we can effectively then have a tremendous number of stale processes running

Reply:
Maybe after a small amount of restarts the log gets pushed to all installed apps' individual logs? So we dont need a way to expose it centrally but is available through all apps

Reply:
> Is there any chance of deno being "restarted" wrongly?

You mean like, unnecessarily? I guess it's possible, but the "restart" procedure attempts to kill the living process before spawning a new one, and if we fail killing the process we just halt. So I'd think the chances are slim

> Maybe after a small amount of restarts the log gets pushed to all installed apps'/affected app's individual log?

I can add a new warn log if the restart call happens after a threshold of 3 times

Reply:
What happens if a user attempts to "enable" an app while it's being restarted? Could it cause 2 attempts of start, and 2 loops of "app restarting"?

I think we can have a special log of "app restarting" every time it happens so users can also check on them, and we as well. Cause if an app is restarting 2 times every some time, some actions may not work but since we're not passing the threshold, we won't see what's going on.

Also, i think these restart logs should (even if we don't show on the UI) have all the information needed about the process failing and a why if possible. wdyt?

Reply:
> What happens if a user attempts to "enable" an app while it's being restarted? Could it cause 2 attempts of start, and 2 loops of "app restarting"?

If a user tries to enable an app while it's being restarted it will fail because the process is not ready to take in requests

Reply:
I've modified the logging a bit. Please @tapiarafael @debdutdeb @KevLehman check it out again